src/main/java/com/github/sylvainlaurent/maven/swaggervalidator/doc/traversal/node/ObjectPropertyRequiredStore.java

```diff
@@ -0,0 +1,25 @@
+package com.github.sylvainlaurent.maven.swaggervalidator.doc.traversal.node;
+
+import static java.util.Collections.synchronizedMap;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ObjectPropertyRequiredStore {
+
+    private static final Map<String, List<String>> requiredObjectPropertiesMap = synchronizedMap(new HashMap<String, List<String>>());
```

Comment:
you'd better use a `java.util.IdentityHashMap` which is made for the use case you have here and is actually safer : the key you create for the Map could have some collisions if 2 objects have the identityHashCode.

Reply:
@sylvainlaurent thanks for the advice, I didn't know about java.util.IdentityHashMap, always good to learn something new ;)